src/gate_detect/src/GateDetectorNode.cpp

```diff
@@ -20,10 +20,7 @@ GateDetectorNode::GateDetectorNode(int argc, char** argv) {
     int cannyLow, houghLinesThreshold, houghLinesMinLength, poleMax,
     lowVertThresh, lowHorThresh, houghLinesMaxLineGap;
     double interpolationConstant1, interpolationConstant2;
```

Comment:
more descriptive names for these would be really helpful

Reply:
Added reference to descriptions